scripts/replace-import.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * This is a jscodeshift transformer that replaces an import statement with another import statement.
+ * It's useful for updating import paths after a module has been moved or renamed.
+ *
+ * Usage:
+ * yarn run jscodeshift --transform=./scripts/replace-import.ts --extensions=js,ts --parser=ts src --originalImportPath=... --originalImportName=... --newImportPath=... --newImportName=...
+ * yarn run jscodeshift --transform=./scripts/replace-import.ts --extensions=jsx,tsx --parser=tsx src --originalImportPath=... --originalImportName=... --newImportPath=... --newImportName=...
```

Comment:
A few of our plain TypeScript files can't be parsed by the TSX parser. We can probably update those and simplify this to one command.